### PR TITLE
Implemented endpoint to return array of topic objects to client

### PR DESCRIPTION
**Pull request by tkbunbury**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe('/api/topics', () => {
+    test('GET:200 sends an array of topic objects to the client', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then((response) => {
+        expect(response.body.topics.length).toBe(3);
```
Comment (nc-ali-abdallah) [inline __tests__/app.test.js:16]: Great job checking the length

File: app.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const app = express();
+const {
+    getTopics,
+
+} = require(`${__dirname}/controller.js`)
+
+app.use(express.json());
```
Comment (nc-ali-abdallah) [inline app.js:8]: Should this be here yet? are we dealing with any sort of request body yet?

File: model.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+selectTopics = async () => {
+    try {
+        const result = await db.query('SELECT * FROM topics;');
+        return result.rows;
+    }
+    catch (error) {
+        console.log(error)
+        throw error;
+    }
```
Comment (nc-ali-abdallah) [inline model.js:13]: We don't really need a catch block here as our controller will catch any errors and pass them onto any error handlers we create

**Review by nc-ali-abdallah — COMMENTED** [review]
Nice job.
We could add an error 500 if we wanted.
Since we know express has a built in 404 handler we are safe to use it but we COULD test that its working.

otherwise great work.

File: model.js (hunk 3 above)
Comment (tkbunbury) [reply model.js:13]: I thought I always need a try block and catch block when using async await syntax? Or is it okay to have try block by itself?

**Pull request merged**